.coderabbit.yml

```
@@ -0,0 +1,180 @@
+# Language-specific settings
+language: "go"
+
+# Review settings
+reviews:
+  # Auto-review settings
+  auto_review:
+    enabled: true
+    
+  # Request changes when critical issues are found
+  request_changes: true
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)